Automattic/woocommerce-payments
# Add risk level information to the Fraud & Risk meta box

Fixes #1225

#### Changes proposed in this Pull Request

This PR aims to add the risk level information to the Fraud & Risk meta box on the order details page. The risk level information provides more information and clarity around the possible risk of the order.

Besides that, an "Adjust risk filters" link was added to the block, allowing merchants to go to the risk filter configuration in an easier way.

| Scenario | Meta box |
|--------|--------|
| Processed with another payment method | ![Screenshot 2024-10-02 at 12 03 31](https://github.com/user-attachments/assets/7e53503c-62ea-47e2-a603-a294e4d6c28f) |
| Order created while Fraud and Risk was disabled | ![Screenshot 2024-10-02 at 11 42 32](https://github.com/user-attachments/assets/f3b6bfb1-170e-45e9-a5a4-4358d4e31042) |
| Order created before this PR | ![Screenshot 2024-10-02 at 11 42 03](https://github.com/user-attachments/assets/b043d5d7-9c9a-4205-a023-a9354d811bfb) |
| Normal risk level | ![Screenshot 2024-10-02 at 11 43 02](https://github.com/user-attachments/assets/05bf80fa-9424-44eb-b8f2-32635d5849a4) |
| Elevated risk level | ![Screenshot 2024-10-02 at 11 41 49](https://github.com/user-attachments/assets/a92ff53c-3a49-4c9e-86a6-9b972e7e850c) | 
| Blocked order | ![Screenshot 2024-10-02 at 11 41 26](https://github.com/user-attachments/assets/2a8ca51f-6763-483f-a957-5c7aa909d3a7) |









#### Testing instructions





* Checkout to this branch
* Unit tests should be passing
* Run `npm run listen`
* On your shop, buy a product using the `4242424242424242` card
* Check the latest order and confirm that the "Fraud & Risk" meta box looks like:

![Screenshot 2024-10-02 at 11 43 02](https://github.com/user-attachments/assets/05bf80fa-9424-44eb-b8f2-32635d5849a4)

* On your shop, buy a product using the `4000000000009235` card
* Check the latest order and confirm that the "Fraud & Risk" meta box looks like:

![Screenshot 2024-10-02 at 11 41 49](https://github.com/user-attachments/assets/a92ff53c-3a49-4c9e-86a6-9b972e7e850c)

* On your shop, buy a product using a different payment method (Cash on delivery etc)
* Check the latest order and confirm that the "Fraud & Risk" meta box looks like:

![Screenshot 2024-10-02 at 12 03 31](https://github.com/user-attachments/assets/7e53503c-62ea-47e2-a603-a294e4d6c28f)

* Update your Fraud protection settings
* On your shop, buy a product using the `4242424242424242` card and make sure it gets blocked by your rules
* Check the latest order and confirm that the "Fraud & Risk" meta box looks like:

![Screenshot 2024-10-02 at 11 41 26](https://github.com/user-attachments/assets/2a8ca51f-6763-483f-a957-5c7aa909d3a7)

-------------------


**Post merge**
